src/main/java/webserver/FileExtension.java

```diff
@@ -0,0 +1,48 @@
+package webserver;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public enum FileExtension {
```

Comment:
정적 팩토리 메서드의 네이밍에서도 드러나듯이 이 enum 요소들은 `ContentType`을 의미하는 것 같은데,  enum class의 네이밍도 ContentType으로 수정해보면 어떨까요? 🙂

```suggestion
public enum ContentType {
```

Reply:
해당 피드백 반영했습니다! 680cecc